bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1515,7 +1508,13 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 >lnd/clm2/initdata_map/clmi.FHISTSp.1979-01-01.ARCTICGRIS_ne30x8_mt12_simyr1979_c200806.nc
 </finidat>
 
-<!-- Present day crop spinup at 1/4-degree for high resolution with irrigation on -->
+<!-- 1979 no crop and present day crop spinup at 1/4-degree for high resolution with irrigation on -->
+<finidat hgrid="ne120np4.pg3" maxpft="17"  mask="tx0.1v3" use_cn=".false." use_cndv=".false." use_fates=".false."
+         ic_ymd="19790101"  sim_year="1979" do_transient_pfts=".false." use_excess_ice=".false."
+         ic_tod="0" glc_nec="10" use_crop=".false." irrigate=".true."
+         lnd_tuning_mode="clm6_0_cam7.0"
+>lnd/clm2/initdata_map/clmi.F2000.2000-01-01.ne120pg3_mt13_simyr2000_c200728.nc
```

Comment:
Make this block same as the next except that it starts in 1979. This makes it more permissive, i.e. it can pick up this finidat for Sp and for BgcCrop.

Reply:
While making the recommended change, I discovered a small typo that could explain previous failures to pick up the wanted finidat. So check the results of the latest test-suites on derecho. If all is good, commit the changes, push and then submit aux_clm on izumi.

Reply:
Good news so far:
aux_clm and ctsm_sci worked the same as before, no better, no worse. In the latter test-suite, two RXCR* are still in progress.